src/python/BFinitPassthrough.py

```diff
@@ -105,38 +105,45 @@ def bf_passthrough_init(port, requestedBaudrate, half_duplex=False, reset_to_bl=
     dbg_print("Enabling serial passthrough...")
     dbg_print("  CMD: '%s'" % cmd)
     rl.write(cmd + '\n')
-
-    dbg_print("======== PASSTHROUGH DONE ========")
-
-    if(reset_to_bl):
```

Comment:
this is moved into main function to enable passthrough flashing using USB FTDI adapter.